mRemoteV1/Resources/Help/ui_menus.htm

```diff
@@ -0,0 +1,224 @@
+      <span style="color: crimson;">Red</span> - Anchor to move menu around the interface<br>
+      <span style="color: forestgreen;">Green</span> - The menu items
+    </p>
+  </p>
+  <h2>Quick Reference</h2>
+  <ul>
+    <li><a href="#file_menu"><b>File Menu</b></a> - Contains standard commands for the application</li>
+    <li><a href="#view_menu"><b>View Menu</b></a> - Menu for additional dialogs for mRemoteNG</li>
+    <li><a href="#tools_menu"><b>Tools Menu</b></a> - Additional tools that can be used and triggered in mRemoteNG</li>
+    <li><a href="#help_menu"><b>Help Menu</b></a> - Get more information for the application</li>
+  </ul>
+  <!-- Introduction - #end -->
+
+  <!-- File menu - #start -->
+  <h1 id="file_menu">File Menu</h1>
+  <hr>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>New Connection</td>
+      <td>Will add a new connection to the Connections dialog after where the cursor is positioned.</td>
+    </tr>
+    <tr>
+      <td>New Folder</td>
+      <td>Add a new folder in the Connections dialog tree where the cursor is positioned.</td>
+    </tr>
+    <tr>
+      <td>New Connection File</td>
+      <td>Create a new connection file. Dialog will come up ask about: filename and where to place the new connection file.</td>
+    </tr>
+    <tr>
+      <td>Open Connection File</td>
+      <td>
+        Open a connection file. Dialog comes up asking about which file to open. For security reasons, this also shows a dialog
+        to ask if you want to save the current file before continuing.
+      </td>
+    </tr>
+    <tr>
+      <td>Save Connection File</td>
+      <td>Saves the current connection file open. If you are using a SQL server connection instead it will send a save to the SQL server.</td>
+    </tr>
+    <tr>
+      <td>Save Connection File As...</td>
+      <td>Saves the current connection file to a specific location on disk.</td>
+    </tr>
+    <tr>
+      <td>Delete...</td>
+      <td>Delete current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Rename</td>
+      <td>Rename current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Duplicate</td>
+      <td>Duplicate current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Reconnect All Open Connections</td>
+      <td>Sends a reconnect to all the open connections in mRmoteNG.</td>
+    </tr>
+    <tr>
+      <td>Exit</td>
+      <td>Exit mRemoteNG application</td>
+    </tr>
+  </table>
+  <!-- File menu - #end -->
+
+  <!-- View menu - #start -->
+  <h1 id="view_menu">View Menu</h1>
+  <hr>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>Add Connection Panel</td>
+      <td>Create a new and empty panel.</td>
+    </tr>
+    <tr>
+      <td>Connection Panels</td>
+      <td>Jump to panel. Useful if you are organised with panels in mRemoteNG to sort out open connections.</td>
```

Comment:
"...Useful if you are organised with panels..." This phrase doesn't work very well, though I'm also not sure what to replace it with. Maybe "Useful if you have many panels...", but then the last part of the sentence will need to change as well.

Reply:
For now I will remove that line a think of something better for the sphinx-doc commit of docs